model/installation_filestore.go

```diff
@@ -0,0 +1,79 @@
+package model
+
+import (
+	"github.com/mattermost/mattermost-cloud/internal/tools/aws"
+	log "github.com/sirupsen/logrus"
+
+	mmv1alpha1 "github.com/mattermost/mattermost-operator/pkg/apis/mattermost/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	// InstallationFilestoreMinioOperator is a filestore hosted in kubernetes via the operator.
+	InstallationFilestoreMinioOperator = "minio-operator"
+	// InstallationFilestoreAwsS3 is a filestore hosted via Amazon S3.
+	InstallationFilestoreAwsS3 = "aws-s3"
```

Comment:
```suggestion
	InstallationFilestoreAWSS3 = "aws-s3"
```

Should we capitalize AWS?

Reply:
I did initially, but found it a bit hard to read so wasn't sure. I am fine with either version.

Reply:
0/5 just saw us capitalizing other acronyms in other places